language/ir-testsuite/tests/discovery/get_discovery_index.mvir

```diff
@@ -0,0 +1,20 @@
+import 0x0.LibraSystem;
```

Comment:
We probably shouldn't be adding new IR tests except for the bytecode verifier. Perhaps try porting this with @tnowacki's tool:

`cargo run -p move-lang --bin  ir-test-translation -- discovery`

and use the same trick with the new test below? Bonus points for porting the old discovery/validator set tests...